_rest_routes.inc.php

```diff
@@ -624,14 +627,30 @@
     "GET /fhir/Patient" => function (HttpRestRequest $request) {
         if ($request->isPatientRequest()) {
             // only allow access to data of binded patient
-            $return = (new FhirPatientRestController())->getOne($request->getPatientUUIDString());
+            // in Patient context you still have to return a bundle even if it is just one resource.
+            $params = $_GET;
+            $params['_id'] = $request->getPatientUUIDString();
+            $return = (new FhirPatientRestController())->getAll($params);
```

Comment:
TODO for me:
In my ongoing PR, will add a second param in this getAll with puuid to ensure no way to get other patient info

Reply:
this is almost completed. i'll plan to bring in that separate PR in after this PR is in codebase